src/com/company/FootballMatch.java

```diff
@@ -19,21 +19,21 @@ public int getInt(){//валидатор ввода данных
         return num;
     }
 
-    //метод подсчёта исхода матча RealOutcome 1 и 2 - реальный счёт, Forecast 1 и 2 - прогноз пользователя
+   
     public int MatchForecast(int RealOutcome1,int RealOutcome2,int Forecast1, int Forecast2){
-        if((RealOutcome1 == Forecast1 && RealOutcome2 == Forecast2))//пользователь угадал, исход = 2
+        if((RealOutcome1 == Forecast1 && RealOutcome2 == Forecast2))
```

Comment:
ternary operate have another structure. For example: return (RealOutcome1 == Forecast1 && RealOutcome2 == Forecast2) ? 2 : 1 
https://www.baeldung.com/java-ternary-operator